examples/BertNewsClassification/README.md

```diff
@@ -1,76 +1,144 @@
+3. Deployment of the  model in TorchServe.
+
+Torchserve deployment plugin has the ability to detect and add the `requirements.txt` and the extra files. And hence, during the
+mar file generation, TorchServe automatically bundles the `requirements.txt`and extra files along with the model.
+
+
 
 ## Package Requirement
 
 Install the required packages using the following command
 
 `pip install -r requirements.txt`
 
-## Installing Deployment plugin
 
-Run the following commands to install deployment plugin
+### Running the code
+To run the example via MLflow, navigate to the `mlflow-torchserve/examples/BertNewsClassification_E2E` directory and run the command
 
-`python setup.py build`
-`python setup.py install`
+```
+mlflow run .
+```
+
+This will run `news_classifier.py` with the default set of parameters such as  `--max_epochs=5`. You can see the default value in the `MLproject` file.
+
+In order to run the file with custom parameters, run the command
 
-## Generating model file (.pt)
+```
+mlflow run . -P max_epochs=X
+```
 
-This example illustrates, storing additional artifacts such as `requirements.txt` and bert vocabulary `bert_base_uncased_vocab.txt`
-along with the model.
+where `X` is your desired value for `max_epochs`.
 
-`requirements_file` and `extra_files` argument in `mlflow.pytorch.log_model` facilitates to store these additional artifacts along
-with the model into mlflow.
+If you have the required modules for the file and would like to skip the creation of a conda environment, add the argument `--no-conda`.
 
-This example performs news classification using Hugging Face bert model and requires `transformers` package to train and predict.
-`trasnformers` package is added as a requirement in `requirements.txt` and stored along with the model using requirement argument.
+```
+mlflow run . --no-conda
 
-Torchserve deployment plugin has the ability to detect and add these `requirements.txt` and the extra files. And hence, during the
-mar file generation, torchserve automatically bundles the `requirements.txt`and extra files along with model.
+```
 
 Note: The arguments `requirements_file` and `extra_files` in `mlflow.pytorch.log_model` are optional.
 
-Run the `news_classifier.py` script which will fine tune the model based on play store review comments. 
+Run the `news_classifier.py` script which will fine tune the model based on the news dataset.
 
 By default,  the script exports the model file as `bert_pytorch.pt` and generates a sample input file `input.json`
 
-Command: 
+Command:
+
+### Passing custom training parameters
+
+The parameters can be overridden via the command line:
+
+1. max_epochs - Number of epochs to train models. Training can be interrupted early via Ctrl+C
+2. num_samples -Number of input samples required for training
+3. tracking-uri -Address of the tracking server
+
+
+For example:
+```
+mlflow run . -P max_epochs=5 num-samples=50000
```

Comment:
@karthik-77  the command is missing -P for the num-samples and also max_epochs needs to be changed to epochs, as in the news_classifier.py it is been defined as epochs. Please consider changing this in the script or REAMDE. Also in the MLPorject file line 9 might need to change num_samples to num-samples.

Reply:
Made all the changes suggested.